lib/test_queue/iterator.rb

```diff
@@ -64,6 +65,7 @@ def each
       end
     rescue Errno::ENOENT, Errno::ECONNRESET, Errno::ECONNREFUSED
     ensure
+      $0 = procline
```

Comment:
> Setting it back to the default as we exit this method seems like the right thing to do;

I'd agree, but I thought this process will die very, very shortly thereafter? I hadn't realized it would stick around.